kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -1243,7 +1244,9 @@ pipe cmd file args = do
     runExternalProcess pipeOut exec str = do
         (maybeInput, maybeOutput, _, _) <- createProcess' exec
         let outputFunc = maybe putStrLn hPutStr maybeInput
```

Comment:
I realize this isn't directly related to the issue at hand but it should be cleaned up while we have the opportunity. `createProcess'` uses `CreatePipe` for both `std_in` and `std_out` so the input and output handles will always be `Just` (see [System.Process](https://hackage.haskell.org/package/process-1.6.13.2/docs/System-Process.html#v:createProcess)). Therefore `outputFunc` will always be `hPutStr`. So we can replace `outputFunc` with `hPutStr hOut` everywhere in the function (line 1249) and just write `(Just hIn, Just hOut, _, _) <- createProcess' exec`. We can also then cleanup the `case maybeOutput` to:
```
output <- liftIO $ hGetContents hOut
modifyIORef pipeOut (appReplOut . AuxOut $ output
```
No need to handle the `Nothing` case anymore.

Reply:
```haskell
(Just hIn, Just hOut, _, _) <- createProcess' exec
X.handle
  (\(X.SomeException e) -> hPutStrLn stderr (displayException e))
  (hPutStr hIn str)
output <- liftIO $ hGetContents hOut
modifyIORef pipeOut (appReplOut . AuxOut $ output)
```

Reply:
Perhaps using `catch` is tidier:
```haskell
hPutStr hIn str `catch` \X.SomeException e -> hPutStrLn stderr (displayException e)
```